testing/python/fixture.py

```diff
@@ -4046,3 +4046,50 @@ def test_2(fix):
         )
         reprec = testdir.inline_run()
         reprec.assertoutcome(passed=2)
+
+
+class TestFixtureParametrizationAndAutouseUsages(object):
+    """Make sure parameterized autouse fixtures tigger re-execution of other
+    fixtures for the same scope.
+     test_module (module)
+    ├── TestIt["a"] (class)
+    |   └── before_param
+    |       └── param_fix (is parametrized)
+    |           └── other_fix
+    |               └── test_stuff (function)
+    └── TestIt["b"] (class)
+        └── before_param
+            └── param_fix (is parametrized)
+                └── other_fix
+                    └── test_stuff (function)
+    """
+
+    def test_combined_parametrizations(self, testdir):
+        testdir.makepyfile(
```

Comment:
I just checked and this test also passes on `master` without your modifications, so we will need another test that exercises the new behavior better.